lab_3_keywords_textrank/main.py

```diff
@@ -81,7 +91,7 @@ def preprocess_text(self, text: str) -> tuple[str, ...]:
             tuple[str, ...]
                 clean lowercase tokens with no stop-words
         """
-        pass
+        return self._remove_stop_words(self._clean_and_tokenize(text))
```

Comment:
it is not clear. Save the result of `self._clean_and_tokenize(text)` to a variable and the use it to call `_remove_stop_words`

Reply:
done